### PR TITLE
Game and store UI

### PR DESCRIPTION
**Pull request by luSteven01**
Adding some touch ups to the UI, and dynamic coloring for readability when going from light or dark mode

### CODE REVIEW

File: app/(site)/temp/all-games/AllGamesDisplay.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import Image from "next/image";
 import Game from "@/types/models/Game";
 import WishlistRow from "@/types/models/WishlistRow";
 import WishListButton from "@/app/ui/components/buttons/WishListButton";
-// import generatePhotoPid from "@/utils/generatePhotoId";
```
Comment (KingSand08) [inline app/(site)/temp/all-games/AllGamesDisplay.tsx:8]: I would not remove this, we might need it for later. We should only remove at the end.

File: app/(site)/temp/all-games/AllGamesDisplay.tsx [hunk 2]
```diff
@@ -20,65 +20,55 @@ export default function AllGamesDisplay({
     uid,
     wishlist,
 }: AllGamesDisplayProps) {
-
     return (
-        <div className="space-y-4">
+        <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6 p-6">
             {games.length > 0 ? (
-                games.map((game) => {
```
Comment (KingSand08) [inline app/(site)/temp/all-games/AllGamesDisplay.tsx:27]: Same thing here: "I would not remove this, we might need it for later. We should only remove at the end."

File: app/(site)/temp/all-games/AllGamesDisplay.tsx [hunk 3]
```diff
@@ -20,65 +20,55 @@ export default function AllGamesDisplay({
...
-                            </div>
-
-                            {/* Game Details */}
-                            <div className="ml-4 flex-grow">
-                                <h2 className="text-xl font-bold">{game.title}</h2>
-                                <p className="text-gray-400">Developer: {game.developer}</p>
-                                <div className="mt-2 text-sm">
-                                    <span>${game.price}</span>
-                                </div>
-                                {/* <div className="mt-1 text-xs text-gray-500">
```
Comment (KingSand08) [inline app/(site)/temp/all-games/AllGamesDisplay.tsx:65]: Same thing here: "I would not remove this, we might need it for later. We should only remove at the end."

**Review by KingSand08 — CHANGES_REQUESTED** [review]
Dont remove  the photo feature, we might still need it for testing later.

File: app/(site)/temp/all-games/AllGamesDisplay.tsx (hunk 3 above)
Comment (luSteven01) [reply app/(site)/temp/all-games/AllGamesDisplay.tsx:65]: added back

File: app/(site)/temp/all-games/AllGamesDisplay.tsx (hunk 2 above)
Comment (luSteven01) [reply app/(site)/temp/all-games/AllGamesDisplay.tsx:27]: added back

File: app/(site)/temp/all-games/AllGamesDisplay.tsx (hunk 1 above)
Comment (luSteven01) [reply app/(site)/temp/all-games/AllGamesDisplay.tsx:8]: added back

**Review by KingSand08 — APPROVED** [review]

**Pull request merged**